index.html

```diff
@@ -24,7 +24,6 @@
     </nav>
   </header>
     
-  
     <div class="first-section">
```

Comment:
- [ ] Please change `div` element with `first-section` class to just section as you used in other places in your code 😉